cli_tools/google-osconfig-agent/packaging/setup_goo.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source packaging/common.sh 
+
+echo "Building package"
+$GO get github.com/google/googet/goopack
+GOOS=windows $GO build -ldflags="-s -w -X main.version=${VERSION}" -o google_osconfig_agent.exe 
```

Comment:
for debuild and rpmbuild, the actual compilation steps are considered the responsibility of the underlying build systems. this is `debian/rules` and the `%build` or `%install` macros, respectively. so in keeping with that, i think the `go build` line should be in the `.goospec` file

Reply:
Ack, this was an attempt to have one place for versioning. I'll move this into the goospec until I can implement variables.